modules/gridBidAdapter.js

```diff
@@ -156,18 +156,68 @@ export const spec = {
     }
 
     if (userId) {
-      userExt = userExt || {};
       if (userId.tdid) {
-        userExt.unifiedid = userId.tdid;
+        userExt = userExt || {};
+        userExt.eids = userExt.eids || [];
+        userExt.eids.push({
+          source: 'adserver.org', // Unified ID
+          uids: [{
+            id: userId.tdid,
+            ext: {
+              rtiPartner: 'TDID'
+            }
+          }]
+        });
+        // userExt.unifiedid = userId.tdid;
```

Comment:
Is the commented code necessary on lines 171, 183, 194, 220? Looks like it's left over from dev?

Reply:
Comments have been removed